master.js

```diff
@@ -120,25 +122,17 @@ class AlAzureMaster {
         this._webAppName = webAppName ? webAppName : process.env.WEBSITE_SITE_NAME;
         this._appFilterJson = process.env.APP_FILTER_JSON ? process.env.APP_FILTER_JSON : '';
         this._appFilterRegex = process.env.APP_FILTER_REGEX ? process.env.APP_FILTER_REGEX : '';
+        
+        //Initialize new SDK
+        this._azureCreds=this.getTokenCredentials();
         if (process.env.FUNCTIONS_EXTENSION_VERSION > '~3') {
-            this._appStats = new AzureAppInsightStats(collectorAzureFunNames);
+            this._appStats = new AzureAppInsightStats(collectorAzureFunNames,this.getTokenCredentials(APPLICATION_INSIGHTS_ENDPOINTS),this._subscriptionId, this._resourceGroup);
```

Comment:
Please refactor, the line is too long.
please add spaces between parameters.

Reply:
resolved